harvester_e2e_tests/integration/test_volumes.py

```diff
@@ -72,3 +74,68 @@ def _wait_for_volume_ready():
         timeout=request.config.getoption('--wait-timeout'))
     assert success, 'Timed out while waiting for volume to be ready.'
     return pvc_json
+
+
+@pytest.mark.volumes
+@pytest.mark.p1
+def test_create_volume_backing_image(api_client, unique_name, opensuse_image,
+                                     wait_timeout, request):
+    """
+    1. Create a new image from URL
+    2. Check that it is created succesffully.
+    3. Create a new volume with the image_id of the previous image
+    4. Check that the new image is created
+    5. Delete image and volume
+    """
+
+    code, image_data = api_client.images.create_by_url(unique_name, opensuse_image.url)
+    if code != 201:
+        raise AssertionError(
```

Comment:
same as:
```python
assert 201 == code, "error messages"
```

Reply:
Good point. I'll change those to be more concise

Reply:
changed.